config/unit_595-20250705T0000/instruments.json

```
@@ -0,0 +1,84 @@
+{
+    "variables": {
+        "instrument_ctd": {
+            "type": "i4",
+            "attributes": {
+                "serial_number": "712-9407",
+                "make_model": "Seabird Slocum Glider Payload CTD",
+                "comment": "Slocum Glider UAF G595 -- pumped CTD",
+                "long_name": "CTD Metadata",
+                "ioos_category": "Identifier",
+                "calibration_date": "2017",
+                "user_calibrated": "",
+                "factory_calibrated": "y",
+                "calibration_report": "https://drive.google.com/file/d/1AyVZJh7Mtlgw97mSCmvIaniQaMPNokT8/view?usp=sharing",
+                "platform": "platform",
+                "type": "instrument",
+                "unit": 1
+            }
+        },
+        "instrument_altimeter": {
+            "type": "i4",
+            "attributes": {
+                "type": "instrument"
+            }
+        },
+        "instrument_gps": {
+            "type": "i4",
+            "attributes": {
+                "type": "instrument",
+                "ioos_category": "Other"
+            }
+        },
+        "instrument_altitude": {
+            "type": "i4",
+            "attributes": {
+                "make_model": "",
+                "type": "instrument"
+            }
+        },
+        "instrument_fluorometer": {
+            "type": "i4",
+            "attributes": {
+                "serial_number": "3933",
+                "make_model": "Wetlabs FLNTUSLC",
+                "comment": "Slocum Glider UAF G595",
+                "long_name": "Wetlabs FLNTUSLC Fluorometer",
+                "calibration_date": "",
+                "factory_calibrated": "y",
+                "calibration_report": "https://drive.google.com/file/d/1DBJAekdh8i_ovytjbjHUHFALe2z8pO7o/view?usp=sharing",
+                "platform": "platform",
+                "type": "instrument",
+                "ioos_category": "Other"
+            }
+        },
+        "instrument_dissolved_oxygen": {
+            "type": "i4",
+            "attributes": {
+                "serial_number": "",
+                "make_model": "",
+                "comment": "",
+                "long_name": "",
+                "calibration_date": "",
+                "factory_calibrated": "",
+                "calibration_report": "",
+                "platform": "platform",
+                "type": "instrument"
+            }
+        },
+        "instrument_acoustic_monitoring": {
+            "type": "i4",
+            "attributes": {
+                "serial_number": "",
+                "make_model": "DMON",
+                "comment": "WHOI manufactured",
+                "long_name": "",
+                "calibration_date": "",
+                "factory_calibrated": "",
+                "calibration_report": "",
+                "platform": "",
+                "type": ""
```

`instrument_acoustic_monitoring` appears to be new (was in provided metadata but doesn't exist in the previous 2022 deployment).

Not sure if the data provider left these fields empty on purpose or if it's safe to assume it should be like this:
```suggestion
                "platform": "platform",
                "type": "instrument"
```

---

Your suggested change is correct.